tests/framework/Optimizers/GeneticAlgorithms/tests

```diff
@@ -384,4 +384,14 @@
      rel_err = 0.001
     [../]
   [../]
+
+  [./GAwithEnsembleModelIncludingCode]
+    type = 'RavenFramework'
+    input = 'continuous/unconstrained/test_ensembleModel_withGA_Code_and_Functions.xml'
+    [./csv]
+     type = UnorderedCsv
```

Comment:
@alfoa FYI, you need to change 'UnorderedCsv' to 'UnorderedCSV'. There is an inconsistent naming in the rook test system.